rules-java-archives/addon/src/test/java/org/jboss/windup/rules/apps/java/archives/identify/test/SortedFileArchiveIdentifierTest.java

```diff
@@ -0,0 +1,83 @@
+package org.jboss.windup.rules.apps.java.archives.identify.test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import org.jboss.forge.addon.dependencies.Coordinate;
+import org.jboss.forge.addon.dependencies.builder.CoordinateBuilder;
+import org.jboss.windup.rules.apps.java.archives.identify.SortedFileArchiveIdentifier;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+/**
+ *
+ * @author Ondrej Zizka, ozizka at redhat.com
+ */
+public class SortedFileArchiveIdentifierTest
+{
+
+    public SortedFileArchiveIdentifierTest()
+    {
+    }
+
+
+    @Test
+    public void testGetCoordinateFromSHA1() throws FileNotFoundException
+    {
+        // TODO: I suggest to rename archive-metadata to archive-mapping. It's not metadata.
+        final File file = new File("target/test-nexus-data/central.archive-metadata.txt");
```

Comment:
Yep, as Jess said - I wanted the pivot search to be tested on the data it will work on in production. If something is removed, then we have no choice than to fix the test. But that's easy.

+1 to syncing the versions. Will do.